Lib/test/test_pkgutil.py

```diff
@@ -318,6 +321,36 @@ def test_name_resolution(self):
                 with self.assertRaises(exc):
                     pkgutil.resolve_name(s)
 
+    def test_name_resolution_import_rebinding(self):
```

Comment:
It may be worth a comment here pointing to the tests of Python's import behavior on these same module examples.

Reply:
Both `import` and `pkgutil` tests use the same data, but I am not sure that their behavior match. It is difficult to say what it means to match or not. I only add these tests so that if next time we will change the behavior, it will be seen in the tests.

Reply:
Yes, I agree with the main motivation, to make future behavior changes visible.

I don't think the comment needs to offer a commentary on how they do or don't match, or a value judgment on the behavior. It just may be useful for someone in the future working in this area to know that there are also actual-import tests using the same sample data, to help them refer to the runtime behavior.

Reply:
It's not important, though, if you don't see the need, feel free to just merge.

Reply:
Done.